### ISSUE TITLE
aws-cis-foundation-benchmark-checklist fails

### ISSUE THREAD

**Issue by lang7** [awslabs/aws-security-benchmark#36]
3.4, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12, 3.13, 3.14 Fail checks because of metric /alarm / subscription missing.

**Comment by Infectsoldier** [comment]
@lang7  3.8, 3.9 and 3.13 is working for me

**Comment by lang7** [comment]
@Infectsoldier  It looks like the LogGroup Metric Filters are not being created - the checks for 3.8, 3.9, and 3.13 are covered by Event Rules / Lambda functions. I don't think aws-cis-foundation-benchmark-checklist.py checks for this ?

**Comment by etendards** [comment]
@lang7 you are correct.  I had to build my own Metric Filters and Alarms to satisfy that check with the python code.  Same for 3.6 and 3.7, still getting false positives.

**Comment by Infectsoldier** [comment]
@etendards you can check the fix for 3.6 and 3.7 in my PR

**Comment by Infectsoldier** [comment]
@lang7 i've create all log groups and metrics using aws cli, not cloudformation template

**Comment by HenrikJaySmith** [comment]
Will look into this. The log group metrics/filters are mapped to the CIS benchmark, will verify against the CFn template.

**Comment by asagage** [comment]
Seems like the py script is correct but the cfn template is deviating from the benchmark.  Although the event rules might be an acceptable alternative to creating a metric/filter + alarm for certain use cases, it isn't really fulfilling the requirement as written.  @lcymburajr can you PR our cfn template changes?

**Comment by riponbanik** [comment]
I have fixed issues with script for 3.2 and added missing checks in template - sorry i did not find any way to create new branch and make pull request here.

https://github.com/riponbanik/aws-cis-foundation-framework

**Comment by riponbanik** [comment]
I have fixed 3.2 pattern issue and updated cfn template with additional checks. Planning to put sam template as well there to create lambda functions and cloudwatch event rule.

https://github.com/riponbanik/aws-cis-foundation-framework
